package-lock.json

```diff
@@ -7384,12 +8298,29 @@
         "path-is-absolute": "^1.0.0"
       }
     },
-    "glob-to-regexp": {
-      "version": "0.4.1",
-      "resolved": "https://registry.npmjs.org/glob-to-regexp/-/glob-to-regexp-0.4.1.tgz",
-      "integrity": "sha512-lkX1HJXwyMcprw/5YUZc2s7DrpAiHB21/V+E1rHUrVNokkvB6bqMzT0VfV6/86ZNabt1k14YOIaT7nDvOX3Iiw==",
-      "dev": true
-    },
+    "glob-parent": {
+      "version": "3.1.0",
```

Comment:
Can you look into if we can upgrade this dependency without webpack 5.x? The dependabot was complaining about glob-parent versions older than 5.1.2